src/test/java/org/opensearch/knn/ODFERestTestCase.java

```diff
@@ -138,8 +138,8 @@ protected void wipeAllODFEIndices() throws IOException {
 
             for (Map<String, Object> index : parserList) {
                 String indexName = (String) index.get("index");
-                if (indexName != null && !".opendistro_security".equals(indexName)) {
-                    adminClient().performRequest(new Request("DELETE", "/" + indexName));
+                if (indexName != null && !".opendistro_security".equals(indexName) && !indexName.matches("knn-bwc-(.*)")) {
```

Comment:
We need to call out somewhere that if you want an index to survive during upgrade, it needs to be prefixed with "knn-bwc-". I actually think it'd be good to add this prefix as a constant in a TestUtil file somewhere.

Reply:
Updated